Framework/Core/src/runDataProcessing.cxx

```diff
@@ -2248,8 +2248,10 @@ int doMain(int argc, char** argv, o2::framework::WorkflowSpec const& workflow,
 
   // We sort dataprocessors and Inputs / outputs by name, so that the edges are
   // always in the same order.
+  // We put the output proxy as last device, to make sure we do not need to copy
+  // messages we are sending somewhere else.
   std::stable_sort(physicalWorkflow.begin(), physicalWorkflow.end(), [](DataProcessorSpec const& a, DataProcessorSpec const& b) {
-    return a.name < b.name;
+    return std::make_pair((int)(a.name == "output-proxy"), a.name) < std::make_pair((int)(b.name == "output-proxy"), b.name);
```

Comment:
I think it is called "dpl-output-proxy" by default, unless you plan to change that as well:
https://github.com/AliceO2Group/AliceO2/blob/707fb83a6cfc47d386d5c363c4206865559af5a3/Framework/Utils/src/dpl-output-proxy.cxx#L25

Reply:
Nice catch. Thanks.